src/routes/(console)/project-[project]/auth/templates/+page.svelte

```diff
@@ -105,7 +103,7 @@
         </div>
     </div>
 
-    {#if !$project.smtpEnabled}
+    {#if !data.project?.smtpEnabled}
```

Comment:
```suggestion
    {#if !data.project.smtpEnabled}
```

using `data` we can be sure the project is always there